### PR TITLE
chore(release): 0.27.0

### PR DESCRIPTION
**Pull request by campionfellin**
*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,28 @@
...
+### Features
+
+* **examples:** Elasticsearch query using CDK8s+ and CRD ([#281](https://github.com/awslabs/cdk8s/issues/281)) ([3be1a96](https://github.com/awslabs/cdk8s/commit/3be1a96b1801f2e23673b454e0a0b59cf7f1b6e5))
+* **lib:** omit duplicate components in generated names ([#258](https://github.com/awslabs/cdk8s/issues/258)) ([473b5ef](https://github.com/awslabs/cdk8s/commit/473b5ef7f442b932e9c4c4356945b91e8e1bc4e4))
+
+
+### Bug Fixes
+
+* **cli:** Conform python and java package names to language standards (no hyphens) ([#283](https://github.com/awslabs/cdk8s/issues/283)) ([f0b33c0](https://github.com/awslabs/cdk8s/commit/f0b33c09e3f75cfc7e230605a9bb11d506ea4018))
+* java importing crd fails ([#257](https://github.com/awslabs/cdk8s/issues/257)) ([f0ef3b4](https://github.com/awslabs/cdk8s/commit/f0ef3b467969a90efcbfc663353565c8639bb5bd))
```
Comment (iliapolo) [inline CHANGELOG.md:25]: Should this also be prefixed with `cli:`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,28 @@
...
+
+
+### ⚠ BREAKING CHANGES
+
+* **lib:** resource names will now be rendered differently, omitting adjacent duplicate components.
+
+Signed-off-by: campionfellin <campionfellin@gmail.com>
+
+*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*
+
```
Comment (iliapolo) [inline CHANGELOG.md:15]: Remove this

File: CHANGELOG.md (hunk 1 above)
Comment (campionfellin) [reply CHANGELOG.md:25]: This is auto-generated by [standard-version](https://www.npmjs.com/package/standard-version) from our bump script: https://github.com/awslabs/cdk8s/blob/master/tools/bump.sh

I should have added the scope in my commit message here: https://github.com/awslabs/cdk8s/pull/257

File: CHANGELOG.md (hunk 2 above)
Comment (campionfellin) [reply CHANGELOG.md:15]: "Signed-off-by" is also auto-generated when I [sign](https://git-scm.com/book/en/v2/Git-Tools-Signing-Your-Work) my commits.

I think the second part is part of the PR templates we have

File: CHANGELOG.md (hunk 1 above)
Comment (iliapolo) [reply CHANGELOG.md:25]: right, in these cases we just manually update the changelog

File: CHANGELOG.md (hunk 2 above)
Comment (iliapolo) [reply CHANGELOG.md:15]: Yes but its usually not part of our release notes...not sure how it got here. lets remove it manually.

File: CHANGELOG.md (hunk 1 above)
Comment (campionfellin) [reply CHANGELOG.md:25]: Gotcha! Will do that then :)

File: CHANGELOG.md (hunk 2 above)
Comment (campionfellin) [reply CHANGELOG.md:15]: Gotcha! Will remove it!

**Pull request merged**